sdk/src/whirlpool-client.ts

```diff
@@ -335,5 +335,45 @@ export interface Position {
     ataPayer?: Address
   ) => Promise<TransactionBuilder>;
 
-  // TODO: Implement Collect fees
+  /**
+   * Collect fees from this position
+   *
+   * If `positionWallet` is provided, the wallet owners have to sign this transaction.
+   *
+   * @param resolveATA -  if true, add instructions to create associated token accounts for tokenA,B for the destinationWallet if necessary. (RPC call required)
+   * @param updateFeesAndRewards -  if true, add instructions to refresh the accumulated fees and rewards data (default to true unless you know that the collect fees quote and on-chain data match for the "feeOwedA" and "feeOwedB" fields in the Position account)
+   * @param destinationWallet - the wallet to deposit tokens into when withdrawing from the position. If null, the WhirlpoolContext wallet is used.
+   * @param positionWallet - the wallet to that houses the position token. If null, the WhirlpoolContext wallet is used.
+   * @param ataPayer - wallet that will fund the creation of the new associated token accounts
+   * @return the transaction that will collect fees from the position
+   */
+  collectFees: (
+    resolveATA?: boolean,
+    updateFeesAndRewards?: boolean,
+    destinationWallet?: Address,
+    positionWallet?: Address,
+    ataPayer?: Address
```

Comment:
updateFeesAndRewards should come first as it's more relevant to the topic of collectFees. the rest are more standard